### PR TITLE
fix: UWP Build

### PR DESCRIPTION
**Pull request by vatsashah45**
GitHub Issue (If applicable): #

What kind of change does this PR introduce?

- Bugfix

## What is the new behavior?

Fixes UWP Build

## PR Checklist

Please check if your PR fulfills the following requirements:
- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] WinUI
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc)
	- [ ] [Controls Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/controls)
	- [ ] [Extensions Doc Update](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc/helpers)
	- [ ] [controls-styles.md](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/lightweight-styling.md)
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs [hunk 1]
```diff
@@ -1,13 +1,16 @@
-﻿using System;
+﻿// FindFirstDescendent is not available in UWP
```
Comment (Xiaoy312) [inline src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs:1]: in toolkit, use **Get**FirstDescendent which is an any-plat impl
the Find... from uno.ui is uno-only

File: src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Uno.UI.RuntimeTests;
 using Uno.Toolkit.RuntimeTests.Helpers;
 using Uno.Toolkit.UI;
 using Uno.UI.Extensions;
 using Uno.Toolkit.RuntimeTests.Tests.TestPages;
-
-
+ 
+
```
Comment (Xiaoy312) [inline src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs:13]: why is space added on every empty line?

File: src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs (hunk 1 above)
Comment (Xiaoy312) [reply src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs:1]: either case, this seems wrong way to fix the problem?

Comment (Youssef1313) [reply src/Uno.Toolkit.RuntimeTests/Tests/AncestorBindingTests.cs:1]: Small note here, `FindFirstDescendent` from Uno.UI is both Uno + WinAppSDK (but not UWP)

https://github.com/unoplatform/uno/blob/9450602cda0194514512ba9572dfc7f745fdfae0/src/Uno.UI.Toolkit/Uno.UI.Toolkit.Windows.csproj#L31-L36

**Comment by vatsashah45** [comment]
Closing this PR, please refer to [this new PR](https://github.com/unoplatform/uno.toolkit.ui/pull/1265) instead

**Pull request closed without merging**